### PR TITLE
ASC-402: Modify NSTemplateTierSpec to support SA token copy

### PR DESCRIPTION
**Pull request by Kartikey-star**
This PR aims at modifying the NSTemplateTier CRD so as to support functionality of copy SA token or not.

See https://github.com/codeready-toolchain/api/pull/380

### CODE REVIEW

**Comment by alexeykazakov** [comment]
Just a note: Adding the link to the parent API change in the PR description is helpful for CRD PRs

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -38,7 +38,6 @@ objects:
           - toolchainconfigs
           - toolchainstatuses
           - proxyplugins
-          - nstemplatetiers
```
Comment (alexeykazakov) [inline deploy/registration-service/registration-service.yaml:41]: I guess you didn't intend to remove it?

Comment (Kartikey-star) [reply deploy/registration-service/registration-service.yaml]: Yeah restored the file.

**Review by mfrancisc — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
:+1:

**Pull request merged**